File: src/modify.js
const wildlyBiasedReview = (location) => {
  if (location === 'NYC') {
    console.log('THE GREATEST CITY IN THE WORLD');
    console.log('THE CITY THAT NEVER SLEEPS');
    console.log('WOW WHAT A CITY');
  } else {
    console.log('Yea that place is cool I guess');
  }
};

const getWeatherReport = (temperature) => {
  if (temperature > 90) {
    const weatherReport = "It's hot and gross out.";
    console.log(weatherReport);
    console.log("And that's your report!");
  } else if (temperature > 70) {
    const weatherReport = "It's really nice!";
    console.log(weatherReport);
    console.log("And that's your report!");
  } else if (temperature < 32) {
    const weatherReport = "Wow, it's cold out.";
    console.log(weatherReport);
    console.log("And that's your report!");
  }
};

module.exports = {
  wildlyBiasedReview,
  getWeatherReport,
};

File: src/modify.py
def wildly_biased_review(location):
    if location == "NYC":
        print("THE GREATEST CITY IN THE WORLD")
        print("THE CITY THAT NEVER SLEEPS")
        print("WOW WHAT A CITY")
    else:
        print("Yea that place is cool I guess")


def get_weather_report(temperature):
    if temperature > 90:
        weather_report = "It's hot and gross out."
        print(weather_report)
        print("And that's your report!")
    elif temperature > 70:
        weather_report = "It's really nice!"
        print(weather_report)
        print("And that's your report!")
    elif temperature < 32:
        weather_report = "Wow, it's cold out."
        print(weather_report)
        print("And that's your report!")
